client-sdk/ts-web-ledger/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "@oasisprotocol/client-ledger",
```

Comment:
Sorry for nitpicking over the name, but I think it may be more clear if it was named `@oasisprotocol/client-signer-ledger` or `@oasisprotocol/signer-ledger`.

Reply:
sure let's go with client-signer-ledger. it'll be good for making sure the scope doesn't creep to include other non-signing functionality